src/predictor/cpu_predictor.cc

```diff
@@ -511,7 +504,7 @@ class ColumnSplitHelper {
     if (missing_bits_.Check(bit_index)) {
       return node.DefaultChild();
     } else {
-      return node.LeftChild() + decision_bits_.Check(bit_index);
+      return node.LeftChild() + !decision_bits_.Check(bit_index);
```

Comment:
Is this a fix, or the meaning of the bits are reversed?

Reply:
The meaning is reversed to match `common::Decision()`.

Reply:
Could you please the comment for `ColumnSplitHelper`?

Reply:
That makes sense.